.github/workflows/ci-build.yml

```diff
@@ -46,7 +46,7 @@ jobs:
     - name: Install Go
       uses: actions/setup-go@v3
```

Comment:
not related to your PR, but I see we use a mix of `actions/setup-go@v4` and `actions/setup-go@v3`, maybe we could upgrade this too.